specification/devopsinfrastructure/resource-manager/Microsoft.DevOpsInfrastructure/preview/2024-04-04-preview/devopsinfrastructure.json

```diff
@@ -141,34 +141,29 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.DevOpsInfrastructure/locations/{locationName}/usages": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DevOpsInfrastructure/locations/{location}/usages": {
       "get": {
-        "operationId": "SubscriptionUsages_ListByLocation",
+        "operationId": "SubscriptionUsages_Usages",
         "tags": [
           "SubscriptionUsages"
         ],
         "description": "List Quota resources by subscription ID",
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
           },
           {
-            "name": "locationName",
-            "in": "path",
-            "description": "Name of the location.",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-.]*$"
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
           }
         ],
         "responses": {
           "200": {
             "description": "Azure operation completed successfully.",
             "schema": {
-              "$ref": "#/definitions/QuotaListResult"
+              "$ref": "#/definitions/PagedQuota"
```

Comment:
You might want to do what you had with an alias with @friendlyName just to keep the same.

Reply:
Leaving it like this since this is following the types that get inherited, no point in defining our custom names